index.js

```diff
@@ -77,6 +99,11 @@ console.log('hello world')
 
 // TODO: ADD CODE HERE
 
+const myFunction = function() {
+    return `from function with love <3`
+}
```

Comment:
Instead of an anonymous function stored in a variable, this should just be a named function:

```javascript
function myFunction() {
    return `from function with love <3`
}
```